README.md

```diff
@@ -40,6 +40,16 @@ The site is now running at `http://localhost:8000`. Any changes you make to code
 
 Note that any changes you make while running Gatsby will automatically checked with [ESLint](https://eslint.org/), so check your console as you save files.
 
+### Organization
+
+Components live in `src/components` and are organized as follows:
+
+* Visualizations belong in `charts`
+* Components that are used more than once and those that are used across different parts of the website belong in `common`
+* Components that control the layout of the website (i.e. headers and footers) belong in `layout`
+* Components that are only used once or have a defined scope to a particular part of the website (i.e. `StateGrade` has to do with states) belong in their respectie directory in `pages`
```

Comment:
spelling - respective

Reply:
I would break this into a list with the directory first:

```markdown
* `/charts` - Data visualizations
* `/common` - Components that....
```

Reply:
on it -- thank you both!